ibazel/main.go

```diff
@@ -19,12 +19,14 @@ import (
 	"fmt"
 	"os"
 	"strings"
+	"time"
 )
 
 var overrideableBazelFlags []string = []string{
 	"--test_output=",
 }
 
+var debounceDuration = flag.Int("debounce", 100, "Debounce duration in milliseconds")
```

Comment:
What would you think about using a duration flag?

https://golang.org/pkg/flag/#Duration

Reply:
Done. Good catch